.travis.yml

```diff
@@ -11,6 +11,11 @@ before_script:
 script: 
   - xcodebuild -project CSyncSDK.xcodeproj -scheme CSyncSDK-iOS -destination platform='iOS Simulator',name='iPhone 6',OS='10.3' clean test
   - xcodebuild -project CSyncSDK.xcodeproj -scheme CSyncSDK-iOS -destination platform='iOS Simulator',name='iPhone 6',OS='9.3' clean test
+  - xcodebuild -project CSyncSDK.xcodeproj -scheme CSyncSDK-OSX -destination platform='macOS' clean test
+  - swift -version
```

Comment:
just so we can look in the logs and see what version is running, could be useful for debugging later